### PR TITLE
Issue #1705650 by damiankloip, aspilicious: Fixed Convert core variables to the config system.

### PR DESCRIPTION
**Pull request by hellomrcat**
PR for https://github.com/backdrop/backdrop-issues/issues/170

### CODE REVIEW

**Comment by quicksketch** [comment]
Thanks for filing this followup. Looks like we have some other issues with system_rss_feeds_settings() in that it's not actually saving the values because there is no system_rss_feeds_settings_submit() function. Unless I'm missing something, it looks like this form wouldn't work. Could you add this submit handler and test the form to ensure the values are getting saved and add the changes to this PR?

**Comment by hellomrcat** [comment]
@quicksketch I added in the submit handler and everything appears to be working correctly (values are saving)

File: core/modules/system/system.admin.inc [hunk 1]
```diff
@@ -1862,16 +1863,31 @@ function system_rss_feeds_settings() {
     '#description' => t('Global setting for the default display of content items in each feed.')
   );
 
+  $form['#submit'][] = 'system_rss_feeds_settings_submit';
+
```
Comment (quicksketch) [inline core/modules/system/system.admin.inc]: These lines aren't necessary (but don't hurt either), as Backdrop will automatically assign this submit handler based on the function name. Let's just delete these two lines and check things still work.

**Comment by hellomrcat** [comment]
@quicksketch looks like it works without the `$form['#submit'][]` line.

File: core/modules/node/views/views_plugin_row_node_rss.inc [hunk 2]
```diff
@@ -96,7 +96,7 @@ class views_plugin_row_node_rss extends views_plugin_row {
 
     $display_mode = $this->options['item_length'];
     if ($display_mode == 'default') {
-      $display_mode = variable_get('feed_item_length', 'teaser');
+      $display_mode = config_get('system.rss', 'rss_viewmode');
```
Comment (quicksketch) [inline core/modules/node/views/views_plugin_row_node_rss.inc]: This should be `config_get('system.rss', 'rss_limit')`.

**Comment by quicksketch** [comment]
This conversion is good but likely going to be unnecessary after the front page gets converted to a View, and the RSS setting is controlled by the view.

**Comment by quicksketch** [comment]
This was converted as part of another issue and then updated in #486.

**Pull request closed without merging**